### DISCUSSION TITLE
【命令语】BP机器人常用命令语整理

### DISCUSSION THREAD

**Discussion by CoffeeFantasy** [mod-playerbots/mod-playerbots#185] [General]
### **Movement（动作）**		
follow	                                跟随	                                                               Follow master
stay                                         呆在原地                        	                               Stay in place
flee	                                       和主人一起逃跑（忽略其他一切）                 Flee with master (ignore everything else)
d attack my target	               攻击我的目标	                                              Attack my target
d add all loot	                      检查每具尸体和游戏物品是否有战利品	      Check every corpse and game object for loot
grind                                      攻击一切	                                                      Attack anything
### **Stragegy                               （策略）**		
co ?                                        需要密语，了解机器人当前战斗列表	
nc ?	                                      需要密语，了解机器人当前非战斗列表	
dead ?	                              需要密语，了解机器人当前死亡状态下列表	
co +s1,-s2,~s3,?	              任何以“+”符号开头的策略名称都将添加到机器人程序策略列表中，	Add, remove, toggle and show combat strategies
nc +s1,-s2,~s3,?	             “-”-从列表中删除，“                                   	Add, remove, toggle and show non-combat strategies
ds +s1,-s2,~s3,?	              ~”-切换（如果列表中没有此类策略，则添加，否则删除）	Add, remove, toggle and show dead strategies
### **Items（道具）**		
e [item]	                              装备[道具]， [item]链接可以替换为设备集名称（套装？），这样机器人就会激活指定的设备集	
ue [item]	                              脱下[道具]	                                                       Unequip item
u [item]	                              使用[道具]	                                                        Use item
u [item] [target]	              在目标上使用物品（例如在物品上使用宝石）	Use item on target (e.g. use gem on item)
destroy [item]	                      摧毁[道具]                                	Destroy item
[item]	机器人会告诉他有多少物品，以及这个物品对他有多有用（例如，对于一个任务，包括还需要多少物品的信息）	
### **Trading（交易）**		
3g 7s 5c	机器人交易时需要给你3金7银5铜，尝试耳语	
[item]	交易给你[道具]，需要打开交易窗口，并且耳语	
s[item]	机器人会将此物品出售给供应商，[物品]可以替换为*，因此机器人会出售所有灰色（质量差）物品	
b[item]	Bot将向供应商购买此商品	
bank [item]	Bot会将此项目放入银行	
bank -[item]	Bot将从银行提取此项目	
gb [item]	机器人会将此物品放入公会银行	
gb -[item]	机器人将从公会银行提取此物品	
### **Looting（拾取）**		
右键点击物品	当你选择某个东西（或右键单击它）时，它将被添加到机器人战利品列表中。	
strategy（自动策略）：		
nc +loot	机器人会将他们杀死的任何生物添加到其战利品列表中。	
ll gray	他们只会掠夺灰色（劣质）物品。	
ll quest	他们只会掠夺任务物品。	
ll skill	他们将根据自己的技能（草药、采矿或剥皮）掠夺物品	
ll normat	他们会掠夺除BOP（捆绑取货）物品以外的任何东西。	
ll all	他们会掠夺一切。	
ll [item]	将物品添加到“始终战利品”列表中	
ll -[item]	将物品添加到“始终战利品”列表中删除	
### ****Quests（任务）****		
accept [quest]	在选定的任务给予者处接受任务	Accept quest at the selected quest giver
accept *	接受所选任务给予者的所有任务	Accept all quests at the selected quest giver
quests incompleted	列出未完成任务名字	
quests completed	列出完成任务名字	
drop [任务名]	放弃[任务]	Abandon quest
r [item]	选择任务奖励	Choose quest reward
quests	查看任务情况	Show quest summary
[quest]	 显示任务和目标状态	Show quest and objectives status
q[quest]	机器人将通过目标告知指定的任务状态和进度	
talk	与选定的NPC对话（完成任务）	Talk to the selected NPC (to complete a quest)
u [game object]	使用游戏对象（使用los命令获取游戏对象链接）	Use game object (use los command to obtain the game object link)
### **Misc（其他）**		
los	登记游戏对象，物品，生物和NPC机器人可以看到	Enlist game objects, items, creatures and NPCs bot can see
stats	显示统计摘要（库存、黄金、xp等）	Show stat summary (inventory, gold, xp, etc.)
leave	离开队伍	Leave party
trainer	显示机器人可以从选定的培训师那里学到什么	Show what bot can learn from the selected trainer
trainer learn	向选定的培训师学习	Learn from the selected trainer
spells	显示机器人的法术	Show bot's spells
cast [spell]	使用法术	Cast the spell
home	在选定的旅店老板处安家	Set home at the selected innkeeper
summon	召唤机器人	Summon bot at the inn
release	死后释放灵魂	Release spirit when dead
revive	附近有灵魂医疗者时复活	Revive when near a spirit healer
### **There are some of master's actions bot can react. For example:		
机器人可以对主人的一些动作做出反应。例如：**		
### **Action（你的动作）	Bot reaction（机器人动作）**
accept a quest	你接受任务，机器人也会接受任务	Bot will accept it as well
talk to a quest giver	你与任务给予者交谈，机器人将交出他完成的任务	Bot will turn in his completed quests
use meeting stone	你使用集合石，机器人帮你拉人	Teleport using the stone
start using game object and interrupt	不知道啥意思	Use the game object
open trade window	打开交易窗口，机器人显示库存并可以交易	Show inventory and start trading
invite to a party/raid	邀请机器人入队，机器人接受	Accept the invitation
start raid ready check	团队检测，机器人点准备	Tell his ready status
mount/unmount	你装载/卸载，机器人也装载/卸载	Mount/unmount as well
go through a dungeon portal	机器人会跟你一起进副本门	Follow into the dungeon
### **HINT: you can create key-binded macros for any of this command for quick usage, e.g.		
提示：为了快速使用，您可以为任何一个命令创建键绑定宏，例如。**		
### **Macro（宏）**
/p follow	跟随我	Follow me
/p stay	停在原地	Stay
/p flee	拉回	Pull back
/p d attack my target	攻击我的目标	Attack my target
/p @tank d attack my target	攻击我的目标（只有坦克职业）	Attack my target (for tanks only)
/p co ~passive,?	切换主动/被动模式（辅助或忽略）	Toggle active/passive mode (assist or ignore)
/p d add all loot	拾取所有东西	Loot everything
/pcan be replaced with/rif you are in raid group.		
/如果你团队中，/p 可以替换为/r		
To learn about other commands tell bot someting invalid and it will enlist all the commands and stategies available.		
要了解其他命令，请告诉机器人某些操作无效，它将登记所有可用的命令和状态。		
				
.bot add name1,name2,etc	.bot添加名称1、名称2等	其中name1、name2等是您想要成为机器人的公会角色的逗号分隔列表。
.bot remove name1,name2,name3	.bot删除名称1、名称2、名称3	
**使用以下命令在组中快速添加/删除所有机器人**		
.bot add *	添加所有	
.bot remove *	删除所有	
**要快速提升您的机器人程序级别和设备，请执行以下命令**		
.bot init=rare name1,name2	这会将指定的机器人级别更改为与角色相同的级别。机器人将学习该级别训练师的所有法术和能力。此外，机器人的设备将被随机分配给一些稀有物品（“稀有”可以改为“稀有” "rare"或“史诗”"epic"）。此命令仅适用于随机机器人程序。

**Comment by CoffeeFantasy** [comment]
文章来源：[https://ike3.github.io/mangosbot-docs/)](url)
如有问题，可进行修改，欢迎讨论

**Comment by oiuv** [comment]
good job，不过你这是机器翻译嘛，还不如让AI翻译，我自己也写了个使用教程在知乎：https://zhuanlan.zhihu.com/p/679421224
